requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy==1.14
-sympy==1.1
```

Comment:
the ``==`` was actually a hack given that 1.12 wasn't working. Can we rather have ``>= 1.1``? or does turning ``dot`` into ``*`` prevent this? I see travis is still having, so you can easily check this with another round of CI

Reply:
Not sure ,lemme try

Reply:
Looks like is fine with `>=`